extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -191,31 +198,32 @@ public function execute( $par ) {
 		$output->setPageTitle( $this->msg( 'editaccount-title' )->plain() );
 
 		$oTmpl = new EasyTemplate( dirname( __FILE__ ) . '/templates/' );
-		$oTmpl->set_Vars( array(
-				'status'	=> $this->mStatus,
-				'statusMsg' => $this->mStatusMsg,
-				'statusMsg2' => $this->mStatusMsg2,
-				'user'	  => $userName,
-				'userEmail' => null,
-				'userRealName' => null,
-				'userEncoded' => urlencode( $userName ),
-				'user_hsc' => htmlspecialchars( $userName ),
-				'userId'  => null,
-				'userReg' => null,
-				'isUnsub' => null,
-				'isDisabled' => null,
-				'isAdopter' => null,
-				'returnURL' => $this->getTitle()->getFullURL(),
-				'logLink' => Linker::linkKnown(
-					SpecialPage::getTitleFor( 'Log', 'editaccnt' ),
-					$this->msg( 'editaccount-log' )->escaped()
-				),
-				'userStatus' => null,
-				'emailStatus' => null,
-				'disabled' => null,
-				'changeEmailRequested' => null,
-				'editToken' => $user->getEditToken(),
-			) );
+		$oTmpl->set_Vars( [
+			'status'	=> $this->mStatus,
+			'statusMsg' => $this->mStatusMsg,
+			'statusMsg2' => $this->mStatusMsg2,
+			'user'	  => $userName,
+			'userEmail' => null,
+			'userRealName' => null,
+			'userEncoded' => urlencode( $userName ),
+			'user_hsc' => htmlspecialchars( $userName ),
+			'userId'  => null,
+			'userReg' => null,
+			'isUnsub' => null,
+			'isDisabled' => null,
+			'isAdopter' => null,
+			'isFanContributor' => null,
+			'returnURL' => $this->getTitle()->getFullURL(),
+			'logLink' => Linker::linkKnown(
+				SpecialPage::getTitleFor( 'Log', 'editaccnt' ),
+				$this->msg( 'editaccount-log' )->escaped()
+			),
+			'userStatus' => null,
+			'emailStatus' => null,
+			'disabled' => null,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "disabled" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)